scripts/docstrings_checker.py

```diff
@@ -196,3 +197,105 @@ class GoogleDocstring(_check_docs_utils.GoogleDocstring):
     ), flags=re.X | re.S | re.M)
 
     re_yields_line = re_returns_line
+
+
+class ASTDocStringChecker(object):
+    """Checks that docstrings meet the code style."""
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def get_args_list_from_function_definition(cls, function_node):
+        """Extracts the arguments from a function definition.
+        Ignores class specific arguments (self and cls).
```

Comment:
I wrote out a bit of functionality to check for class arg ordering, and learned two things in doing so that make me think this isn't the right place or way to do it (the code is not committed because it broke the linter):

1. Raising errors around the ordering of actual args in the function definition, as opposed to errors in the docstring content, extends the scope of this PR and class definition quite a bit. It doesn't make sense conceptually to have these two checks in the same place. 

2. Implementing this check found two instances where `cls` is used as a regular arg in addition to the `self` class arg in a function definition (which broke the rule of "class arguments must be in the 0th position"). I could change the arg names, but I am wary of breaking something if I do. 
`extensions/objects/models/objects_test.py --> Func check_normalization: Class arg must be first: cls`
`extensions/rich_text_components/components_test.py --> Func check_validation: Class arg must be first: cls`

What do you think?

Reply:
1. Oh, that's a good point -- my mistake. I agree that that is better tackled for a separate PR. Could you file an issue please?

2. I think changing arg names is fine (and the right thing to do) as long as it's done carefully. E.g. in the first case we could rename cls to object_class and in the second it could be rte_component_class. (I don't think you're in danger of breaking anything, since these are test methods and the args are scoped to just those methods.)

Thanks for checking!

Reply:
I filed an issue here: https://github.com/oppia/oppia/issues/5876

(How does this work- do I hit "Resolve conversation" now, or do you?)

Reply:
Thanks! Feel free to go ahead and hit "Resolve conversation" once you've resolved a comment and replied to it (e.g. with "Done" or equivalent).

Reply:
Done